src/Silences/Types.elm

```diff
@@ -90,3 +32,20 @@ nullDuration =
 nullTime : Time
 nullTime =
     Utils.Date.fromTime 0
+
+
+type alias Silence =
+    { id : SilenceId
+    , createdBy : String
+    , comment : String
+    , startsAt : Time
+    , endsAt : Time
+    , duration : Duration
+    , updatedAt : Time
+    , matchers : List Matcher
+    , silencedAlertGroups : ApiData (List AlertGroup)
+    }
+
+
+type alias SilenceId =
```

Comment:
Doesn't this make it more descriptive? Instead of handling a `String` in other places we then handle a `SilenceId`? See ELM [Tutorial](https://www.elm-tutorial.org/en/01-foundations/06-type-aliases.html)